Resources/AddService/IPAddress.json

```diff
@@ -0,0 +1 @@
+{"Device": [{"IPAddress": "437XR1138R2"}, {"IPAddress": "437XR1138R3"}, {"IPAddress": "437XR1138R4"}, {"IPAddress": "183.25.16.12"}, {"url": "http://183.25.16.12:5000/redfish/v1/", "IPAddress": "183.25.16.12"}]}
```

Comment:
The Swordfish emulator should not need to include this file or any other special support for the WEB-UI. The WEB-UI needs to be able to operate without this kind of support from the emulator.